gui/src/main.rs

```diff
@@ -39,37 +39,15 @@ pub enum RequestWindowMode {
     ENVIRONMENT,
 }
 
-#[derive(Serialize, Deserialize)]
-pub struct GuiConfig {
```

Comment:
Remembered that this GuiConfig was unused since all the properties were moved to be there own fields on the main Gui struct. Removing.